test/topology_experimental_raft/test_alternator.py

```diff
@@ -297,14 +299,117 @@ async def check_two_live_nodes():
     response = requests.get(localnodes_request)
     j = json.loads(response.content.decode('utf-8'))
     assert len(j) == 1
-    # Ending the test here will kill both servers. We don't wait for the
-    # second server to finish its long injection-caused bootstrap delay,
-    # so we don't check here that when the second server finally comes up,
-    # both nodes will finally be visible in /localnodes. This case is checked
-    # in other tests, where bootstrap finishes normally - we don't need to
-    # check this case again here.
-    task.cancel()
+
+    # We don't want to wait for the second server to finish its long
+    # injection-caused bootstrap delay, so we won't check here that when the
+    # second server finally comes up, both nodes will finally be visible in
+    # /localnodes. This case is checked in other tests, where bootstrap
+    # finishes normally, so we don't need to check this case again here.
+    # But we can't just finish here with "task" unwaited or we'll get a
+    # warning about an unwaited coroutine, and the ScyllaClusterManager's
+    # tasks_history will wait for it anyway. For the same reason we can't
+    # task.cancel() (this will cause ScyllaClusterManager's tasks_history
+    # to report the ScyllaClusterManager got BROKEN and fail the next test).
+    # Sadly even abruptly killing the servers (with manager.server_stop())
+    # (with the intention to then "await task" quickly) doesn't work,
+    # probably because of a bug in the library. So we "await task"
+    # anyway, and this test takes 2 minutes :-(
```

Comment:
maybe we can put smaller timeout for delay_bootstrap? 

attempt to start ```18:45:43.882 INFO> [dev/topology_experimental_raft.test_alternator.4] starting server at host 127.212.242.38 in scylla-106...```

test finished ```18:45:45.875 INFO> [dev/topology_experimental_raft.test_alternator.4] [ScyllaClusterManager][Task-7184] http://api/cluster/after-test/True```

Reply:
@temichus this test waits until something happens, and needs to delay the bootstrap but *more* than this wait time. In the past I had here 10 seconds for the first wait, and 20 seconds for the boostrap delay. And it was enough. Until it wasn't, on some super-slow-debug-with-aarch64 CI run :-( So I increase the times to 60 seconds and 120 seconds, respectively. I thought it was perfectly fine because we never need to wait for this 120 seconds to finish. The test just kills the server which is in the middle of the boot. Or so I thought. But it doesn't work. And to be honest, I just gave up trying to fix this.

What I **REALLY** want to happen is the test to end abruptly with the second server in the middle of the boot, and the test will just kill this server. But I couldn't figure out a way to do this. If you can figure out a way to do this, great :-) But reducing the 60 seconds and 120 seconds number is not a good idea - it can just make this test flaky on super-slow test runs.